app/controllers/beacon_controller.rb

```diff
@@ -0,0 +1,29 @@
+# simple Beacon implementation according to the GA4GH v0.2 standard
+# see http://dnastack.com/ga4gh/bob/subscribe.html
+# chrom={chromosome}&
+# pos={position}&
+# allele={allele}
+
+class BeaconController < ApplicationController
+  def responses
+    begin
+      @position = params[:pos].to_i
+      @chromosome = params[:chrom]
+      @allele = params[:allele].upcase
+      # get all snps, iterate over them:
+      #if found the allele: return yes & break
+      @snps = Snp.where(position: @position,chromosome: @chromosome)
+      @snps.each do |s|
+        if s.allele_frequency[@allele] > 0
+          render :text => "YES" and return
```

Comment:
Use `&&` instead of `and`.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.